docs/webhooks/index.html

```diff
@@ -318,10 +318,16 @@ <h4 id="preScreenExpired">preScreenExpired</h4>
     testTitle: string,
     candidateEmail: string,
     candidateName: string,
-    expiredReason: string,
+    expiredReason: 'notCertified' | 'testNotTaken', *
```

Comment:
Ah, I should have realized this sooner. Should we be sending "not verified" instead of "not certified" as the expired reason here?

Reply:
I think we should change it on the new web hook. Changing it on the pre existing ones might break customer code. 

So if we change it on the new pre-Screen expired webhook we would be moving away from Certify wording once we sunset Certify.

Reply:
Actually, we could just remove the 'notCertified' option, because that's no longer the case in Pre-Screen Test. The graph is correct and the only way a Pre-Screen Test can expire now is if the candidate doesn't take it.